sklearn/preprocessing/label.py

```diff
@@ -500,46 +547,51 @@ def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
         y_in_classes = in1d(y, classes)
         y_seen = y[y_in_classes]
         indices = np.searchsorted(sorted_class, y_seen)
-        indptr = np.hstack((0, np.cumsum(y_in_classes)))
+        if pos_label != 0:
```

Comment:
is this tested?

Reply:
I thought that is a nice illustration of the functionality, so you suggest to move it to unit test?